tests/test_registry/test_abi_registry.py

```diff
@@ -170,3 +167,31 @@ def test_copying_copies_internal_mappings(registry: ABIRegistry):
         assert isinstance(x.get_decoder('address'), decoding.AddressDecoder)
         assert isinstance(y.get_encoder('address'), encoding.AddressEncoder)
         assert isinstance(y.get_decoder('address'), decoding.AddressDecoder)
+
+
+def test_has_encoder_returns_true(registry: ABIRegistry):
+    assert registry.has_encoder('address')
+
+    registry.register(
+        BaseEquals('address', with_sub=False),
+        encoding.AddressEncoder, decoding.AddressDecoder,
+        label='other_address',
+    )
+
+    with pytest.raises(exceptions.MultipleEntriesFound):
+        assert registry.has_encoder('address')
```

Comment:
Looks like all but one line in this test is a duplicate of `test_has_encoder_raises()`

Reply:
Ahh, yep.  Probably just copied and pasted wrong.